### PR TITLE
update setup.sh and add .idea files to .git ignore

### PR DESCRIPTION
**Pull request by razashtar93**
Added:  pipenv run python manage.py migrate  to setup.sh

Added .idea/ files to .gitignore

deleted:    .idea/workspace.xml from git using:  git rm .idea/workspace.xml -f  command

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -137,3 +137,13 @@ nohup.out
 # AcadeMeData app
 AcadeMeData/.idea/
 
+# AcadeMe app - all of the .idea files that created by pycharm
+AcadeMe/.idea/
```
Comment (Yarboa) [inline .gitignore:141]: As per this 
https://git-scm.com/book/en/v2/Git-Basics-Recording-Changes-to-the-Repository#_ignoring

the following should do the trick for all subdirectories, I think
**/.idea

Although this template show it explicitly
https://github.com/github/gitignore/blob/master/Android.gitignore#L41_L52

Comment (bregman-arie) [reply .gitignore:141]: .idea/* should do the trick

**Comment by Yarboa** [comment]
What about this PR, there was another one in the same subject
https://github.com/redhat-beyond/AcadeMe/pull/51

**Comment by Yarboa** [comment]
Please rebase with upstream

**Pull request closed without merging**